tasks/collector.py

```diff
@@ -242,3 +258,215 @@ def generate(ctx):
                     f.write(content)
 
                 print(f"Updated package name and ensured license header in: {file_path}")
+
+
+GITHUB_API_URL = "https://api.github.com/repos"
+
+
+def fetch_latest_release(repo):
+    url = f"{GITHUB_API_URL}/{repo}/releases/latest"
+    print(f"Fetching latest release from {url} for {repo}")
+    response = requests.get(url)
+    if response.status_code == 200:
+        data = response.json()
+        return data["tag_name"]
+    else:
+        return None
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You should consider using the github API instead of requests (`tasks.libs.ciproviders.github_api`)

Reply:
Agree with Celian, I think relying on the existing Github API should be simpler here. Let us know if you need any help with that